src/components/VideoLike/VideoLike.js

```diff
@@ -0,0 +1,61 @@
+import React, { useEffect } from "react";
+import axios from "axios";
+import { useAuth } from "context/auth-context/auth-context";
+import { Link } from "react-router-dom";
+import { useLike } from "context/like-context/like-context";
+import "./VideoLike.css";
+
+export const VideoLike = () => {
+  const { likedVideos, setLikedVideos } = useLike();
+
+  const { auth } = useAuth();
+  useEffect(() => {
+    (async () => {
+      const response = await axios({
+        method: "get",
+        url: "/api/user/likes",
+        headers: { authorization: auth.token },
+      });
+      setLikedVideos(response.data.likes);
+    })();
+  }, [auth.token, setLikedVideos]);
+
+  const removeLikedVideo = (_id) => {
+    (async () => {
+      const response = await axios.delete(`/api/user/likes/${_id}`, {
+        headers: { authorization: auth.token },
+      });
+      setLikedVideos(response.data.likes);
```

Comment:
does delete api call directly delete and returns new filtered array,i thought it return one particular item ,okeiy that's why i was not able to remove from wishlist while using api call.
Nicee i got to know this.